scripts/run-zowe.template.sh

```diff
@@ -111,7 +107,7 @@ then
   mkdir -p ${STATIC_DEF_CONFIG_DIR}
 
   # Until ui explorers componentised will copy them from the old location
-  cp ${ROOT_DIR}/components/api-mediation/api-defs/* ${STATIC_DEF_CONFIG_DIR}
+  # cp ${ROOT_DIR}/components/api-mediation/api-defs/* ${STATIC_DEF_CONFIG_DIR}
```

Comment:
Please delete this line and the comment above

Reply:
removed